lib/ably/rest/client.rb

```diff
@@ -167,10 +166,10 @@ class Client
       # @return [Ably::Rest::Client]
       #
       # @example
-      #    # create a new client authenticating with basic auth
+      #    # The Ably API key or token string used to validate the client.
       #    client = Ably::Rest::Client.new('key.id:secret')
       #
-      #    # create a new client and configure a client ID used for presence
```

Comment:
Also seems wrong, the example isn't using a `Hash` object

Reply:
It's using Hash or String. Both ways are correct. I have fixed.